### PR TITLE
Changes to Launch Config

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
1. Removes duplicate configs for Lightning and LWC Integration Tests.
2. Adds config for running Lightning and LWC Unit Tests.
3. Renamed configs for easier recognition of Test configs.
4. Reordered Replay Debugger config so all Test configs were together.

### Functionality Before
Screen Shot 2020-04-17 at 8 26 49 AM

### Functionality After
Screen Shot 2020-04-17 at 9 25 49 AM

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Launch Salesforce DX Utils Unit Tests",
```
Comment (sfsholden) [inline .vscode/launch.json:89]: Renamed our Test config launches. This feels much easier to read.

File: .vscode/launch.json [hunk 2]
```diff
@@ -69,14 +57,26 @@
...
-      ],
+      "outFiles": ["${workspaceFolder}/packages/*/out/src/**/*.js"],
+      "sourceMapPathOverrides": {
+        "webpack:///salesforcedx-utils-vscode/./*": "${workspaceFolder}/packages/salesforcedx-utils-vscode/*",
+        "webpack:///*": "*"
+      },
+      "preLaunchTask": "Compile"
+    },
+    {
+      "name": "Launch Replay Debugger adapter",
```
Comment (sfsholden) [inline .vscode/launch.json:71]: Moved this entry so it wasn't in the test section.

File: .vscode/launch.json [hunk 3]
```diff
@@ -159,80 +153,76 @@
...
-        "../../../../test/vscode-integration": "${workspaceFolder}/packages/salesforcedx-vscode-lightning/out/test/vscode-integration/*",
+        "../../../test/unit/": "${workspaceFolder}/packages/salesforcedx-vscode-lightning/out/test/unit/*",
         "webpack:///salesforcedx-utils-vscode/./*": "${workspaceFolder}/packages/salesforcedx-utils-vscode/*",
         "webpack:///*": "*"
       },
+      "preLaunchTask": "Compile",
       "internalConsoleOptions": "openOnSessionStart"
     },
     {
-      "name": "Launch LWC Tests",
```
Comment (sfsholden) [inline .vscode/launch.json:186]: Removed config. Launch Tests - Lightning - VSCode Integration Tests uses the same config.

File: .vscode/launch.json [hunk 4]
```diff
@@ -159,80 +153,76 @@
       "internalConsoleOptions": "openOnSessionStart"
     },
     {
-      "name": "Launch Lightning Tests",
```
Comment (sfsholden) [inline .vscode/launch.json:162]: Removed config. Launch Tests - Lightning - VSCode Integration Tests uses the same config.

File: .vscode/launch.json [hunk 5]
```diff
@@ -159,80 +153,76 @@
...
+        "../../../test/unit/": "${workspaceFolder}/packages/salesforcedx-vscode-lwc/out/test/unit/*",
         "webpack:///salesforcedx-utils-vscode/./*": "${workspaceFolder}/packages/salesforcedx-utils-vscode/*",
         "webpack:///*": "*"
       },
       "preLaunchTask": "Compile",
       "internalConsoleOptions": "openOnSessionStart"
     },
     {
-      "name": "Launch Salesforce DX VS Code LWC VSCode-Integration Tests",
+      "name": "Launch Tests - LWC - VSCode Integration Tests",
```
Comment (lcampos) [inline .vscode/launch.json:225]: The LWC integration tests need to run using `${workspaceFolder}/packages/system-tests/assets/lwc-recipes` as an argument. Take a look at "old" `Launch LWC Tests` config.

Comment (sfsholden) [reply .vscode/launch.json:225]: This config is already using `${workspaceFolder}/packages/system-tests/assets/lwc-recipes`. The only difference with the configs is that `Launch LWC Tests` doesn't have the section `"preLaunchTask": "Compile",`.

Comment (lcampos) [reply .vscode/launch.json:225]: I didn't realize the args part was hidden and had to expand the file.

**Pull request merged**